tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -109,3 +113,27 @@ Verify Quick Starts Work As Expected When At Least One Step Is Skipped
     END
     Run Keyword And Continue On Failure     QuickStart Status Should Be    ${element}      In Progress
     Run Keyword And Continue On Failure     Link Text On QuickStart Card Should Be  element=${element}  exp_link_text=Continue
+
+Verify Quick Starts Work As Expected When Restarted And Left In Between
+    [Arguments]     ${element}
+    QuickStart Status Should Be    ${element}  Complete
+    Open QuickStart Element in Resource Section By Name     ${element}
+    Page Should Not Contain         //article[@id="${element}"]//span[@class="pf-c-label pf-m-green pf-m-outline"]
+    ${count}=   Get The Count Of QuickStart Steps
+    Click Button    //button[@data-testid="qs-drawer-continue"]
+    Wait Until Page Contains Element    //div[@class="pfext-quick-start-content"]
+    ${temp_count}   Set Variable    2
+    FOR     ${index}    IN RANGE    ${temp_count}
+        Click Button    //button[@data-test="Next button"]
+        Run Keyword And Continue On Failure     Mark Step Check As Yes
+        Go To Next QuickStart Step
+    END
+    Close QuickStart From Top     decision=leave
+    Run Keyword And Continue On Failure     Page Should Not Contain QuickStart Sidebar
+    Run Keyword And Continue On Failure     QuickStart Status Should Be    ${element}  In Progress
+    Run Keyword And Continue On Failure     Link Text On QuickStart Card Should Be  element=${element}  exp_link_text=Continue
+    Click Link      //article[@id="${element}"]//a
+    Wait Until Page Contains ELement    //div[@class="pf-c-drawer__panel-main"]     5
+    FOR     ${index}    IN RANGE    ${temp_count}
+        Page Should Contain Element     //div[@class="pfext-quick-start-tasks__list"]/li[${index}+1]//${SUCCESS_STEP}
+    END
```

Comment:
I think this should go to resource file, it can be used for other verification. @bdattoma Thoughts?

Reply:
I would say it is okay to remain here, because this kw is very specific to this use case